addons/website/static/src/js/menu/edit.js

```diff
@@ -72,9 +72,18 @@ var EditPageMenu = websiteNavbarData.WebsiteNavbarActionWidget.extend({
     start: function () {
         var def = this._super.apply(this, arguments);
 
+        let navbar = $('nav#oe_main_menu_navbar').get(0);
+        let stopPropagation = function (event) {
+            event.stopPropagation();
+        };
+
         // If we auto start the editor, do not show a welcome message
         if (this._editorAutoStart) {
-            return Promise.all([def, this._startEditMode()]);
+            navbar.addEventListener('click', stopPropagation, {capture: true});
+            return Promise.all([def, this._startEditMode()]).then(function (result) {
```

Comment:
can't you `await` + put start in async instead of then?
If you can't, at least use es6 arrow